modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/generators.scala

```diff
@@ -0,0 +1,29 @@
+package org.tessellation.rosetta.domain
+
+import org.tessellation.generators.nesGen
+import org.tessellation.schema.generators._
+import org.tessellation.security.hex.Hex
+
+import org.scalacheck.Gen
+
+object generators {
+  val hexGen: Gen[Hex] = nesGen(Hex(_))
```

Comment:
you should reuse existing generators, or specify it in the module corresponding to the class (there: to the Hex class)

Reply:
Thank you. I did reuse existing generators in the rest of the file. Can you point me to an existing generator for `Hex` ? I must have missed it.

Reply:
I'm not sure if there is, but if you cannot find it, most likely there isn't. Anyway, the place of defining it should be corresponding to the Hex class (module) if possible